electron/ironfish/AccountManager.ts

```diff
@@ -296,7 +296,15 @@ class AccountManager
   }
 
   async submitAccount(createParams: AccountValue): Promise<WalletAccount> {
-    const newAccount = await this.node.wallet.importAccount(createParams)
+    const create = {
+      ...createParams,
+      createdAt: {
+        sequence: createParams.createdAt.sequence,
+        hash: Buffer.from(createParams.createdAt.hash),
+      },
+    }
+    const newAccount = await this.node.wallet.importAccount(create)
```

Comment:
https://github.com/iron-fish/node-app/pull/286

Once above PR is merged line 307 will be only change.